examples/set-namespace-kpt-package-context/Kptfile

```diff
@@ -2,6 +2,8 @@ apiVersion: kpt.dev/v1
 kind: Kptfile
 metadata:
   name: example
+  annotations:
+    config.kubernetes.io/local-config: "true"
```

Comment:
Why we need to add this?

Reply:
This is a cleanup. Currently kpt adds this annotation to Kptfile in creation time. e.g. `kpt pkg init` gives 
```yaml
apiVersion: kpt.dev/v1
kind: Kptfile
metadata:
  name: tm
  annotations:
    config.kubernetes.io/local-config: "true"
info:
  description: sample description
```